src/components/settings-menu/DevNotificationsSection.tsx

```diff
@@ -0,0 +1,434 @@
+          tx: state[wallet.address].tx,
+        },
+      }));
+
+      await messaging().unsubscribeFromTopic(
+        `watcher_${wallet.address.toLowerCase()}`
+      );
+      await messaging().subscribeToTopic(
+        `owner_${wallet.address.toLowerCase()}`
+      );
+    });
+  };
+
+  const subscribeTx = async (address: string) => {
+    setNotificationState((state: any) => ({
+      ...state,
+      [address]: { subscription: state[address].subscription, tx: 'on' },
+    }));
+
+    await messaging().subscribeToTopic(`${chainId}_${address.toLowerCase()}`);
+  };
+
+  const subscribeAllTx = async () => {
+    allWallets.forEach(async wallet => {
+      setNotificationState((state: any) => ({
+        ...state,
+        [wallet.address]: {
+          subscription: state[wallet.address].subscription,
+          tx: 'on',
+        },
+      }));
+
+      await messaging().subscribeToTopic(
+        `${chainId}_${wallet.address.toLowerCase()}`
+      );
+    });
+  };
+
+  const unsubscribeTx = async (address: string) => {
+    setNotificationState((state: any) => ({
+      ...state,
+      [address]: { subscription: state[address].subscription, tx: 'off' },
+    }));
+
+    await messaging().unsubscribeFromTopic(
+      `${chainId}_${address.toLowerCase()}`
+    );
+  };
+
+  const unsubscribeAllTx = async () => {
+    allWallets.forEach(async wallet => {
+      setNotificationState((state: any) => ({
+        ...state,
+        [wallet.address]: {
+          subscription: state[wallet.address].subscription,
+          tx: 'off',
+        },
+      }));
+
+      await messaging().unsubscribeFromTopic(
+        `${chainId}_${wallet.address.toLowerCase()}`
+      );
+    });
+  };
+
+  return (
+    <ScrollView>
+      <Box paddingHorizontal="19px" paddingTop="19px">
+        {loading ? null : (
+          <Columns space="8px">
+            {/* ugly undefined to clear TS because this component is not typed */}
+            <MiniButton
+              backgroundColor={colors.blueGreyDark30}
+              color="secondary60"
+              disablePadding={undefined}
+              disabled={undefined}
+              hasLeadingIcon={undefined}
+              height={undefined}
+              hideShadow
+              letterSpacing={undefined}
+              onPress={() => {
+                unsubscribeAll();
+              }}
+              small={undefined}
+              weight={undefined}
+              width={undefined}
+            >
+              All Off
+            </MiniButton>
+            <MiniButton
+              backgroundColor={colors.blueGreyDark30}
+              color="secondary60"
+              disablePadding={undefined}
+              disabled={undefined}
+              hasLeadingIcon={undefined}
+              height={undefined}
+              hideShadow
+              letterSpacing={undefined}
+              onPress={() => {
+                subscribeAllAsWatcher();
+              }}
+              small={undefined}
```

Comment:
i prefer just `expect error`. This doesn't make sense and this patter will be harmful when we will eventually move the component to TS

Reply:
+ maybe there's some setting for this? 
I don't mind what will you end up with as long as this is not redundant like here